core/dbt/utils.py

```diff
@@ -20,6 +20,7 @@
 from dbt.logger import GLOBAL_LOGGER as logger
 from dbt.node_types import NodeType
 from dbt.clients import yaml_helper
+from dbt.ui import printer
```

Comment:
It looks like this triggers an import loop on certain imports (utils -> ui.printer -> tracking -> version -> semver -> utils)! It's unfortunate that your PR triggered this. I think the best thing to do is to move the `from dbt.tracking import InvocationProcessor` out of the top level and into `print_run_end_messages` with a noqa and an explanatory comment like this:
```
def print_run_end_messages(results, keyboard_interrupt: bool = False) -> None:
    # prevent an import loop
    from dbt.tracking import InvocationProcessor
    ...
```